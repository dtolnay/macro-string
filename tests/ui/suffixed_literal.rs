use macro_string_eval::eval;

const _: &str = eval!(concat!("ru"i32, "st"));

const _: &str = eval!(concat!(' 'i32));

const _: &str = eval!(concat!(1i256));

const _: &str = eval!(concat!(1.0f256));

fn main() {}
