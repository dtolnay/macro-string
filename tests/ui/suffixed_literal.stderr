error: unexpected suffix on string literal
 --> tests/ui/suffixed_literal.rs:3:31
  |
3 | const _: &str = eval!(concat!("ru"i32, "st"));
  |                               ^^^^^^^

error: unexpected suffix on char literal
 --> tests/ui/suffixed_literal.rs:5:31
  |
5 | const _: &str = eval!(concat!(' 'i32));
  |                               ^^^^^^

error: unexpected suffix on integer literal
 --> tests/ui/suffixed_literal.rs:7:31
  |
7 | const _: &str = eval!(concat!(1i256));
  |                               ^^^^^

error: unexpected suffix on float literal
 --> tests/ui/suffixed_literal.rs:9:31
  |
9 | const _: &str = eval!(concat!(1.0f256));
  |                               ^^^^^^^
